vcr/vcr.go

```diff
@@ -142,11 +142,24 @@ func (c *vcr) initIndices() error {
 				var leiaParts []leia.IndexPart
 
 				for _, iParts := range index {
-					name := iParts
-					jsonPath := t.ToVCPath(iParts)
-					leiaParts = append(leiaParts, leia.NewJSONIndexPart(name, jsonPath))
+					name := iParts.ConceptPath
```

Comment:
is the `concept` package `go-leia` specific? in that case it could just return the appropriate leia indices (the VCR type has quite a bit of responsibilities now)

Reply:
I'd like to wait for a refactor when we start discussing what the impact is for private VCs + clustering/backups. That could possibly force us to change storage.